### DISCUSSION TITLE
Quick start guide - viewer remains blank

### DISCUSSION THREAD

**Discussion by Johannes-repos** [iTwin/community#51] [General]
Hello,

when using the iTwin viewer at 
https://developer.bentley.com/tutorials/web-application-quick-start/#2-get-the-code

the viewer remains blank with no iModel visible. Same iModel is visible in Infrastructure Cloud.

![image](https://github.com/user-attachments/assets/795a79ee-2016-49ca-a6fd-da76ae8ec29a)

![image](https://github.com/user-attachments/assets/7c71081d-04d2-4b95-b003-08fe689ee324)

A viewer build a few days ago still works, so this must be a recent change.

Also, how to get rid of the Sass deprecation warnings?

![image](https://github.com/user-attachments/assets/008c0308-ab20-4b46-b28f-7bf99bf390ac)

Thank you

**Comment by ben-polinsky** [comment]
Hi,

Any errors in the chrome dev console or failed network requests? What versions of `@itwin/core-` packages are you using?

**Comment by aruniverse** [comment]
Hi Johannes, we are aware of the issue and will be deploying a fix to our visualization backend.

**Reply by aruniverse** [reply]
Sorry forgot to respond, but this should be resolved

**Reply by Johannes-repos** [reply]
Works! Thanks a ton for fixing this so quickly.
